Mathlib/Topology/Algebra/ClopenNhdofOne.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2024 Nailin Guan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan, Yi Song, Xuchun Li
+-/
+import Mathlib.GroupTheory.Index
+import Mathlib.Topology.Algebra.ClosedSubgroup
+import Mathlib.Topology.Algebra.OpenSubgroup
+/-!
+# Open normal subgroup in a clopen neighborhood of One
+This file proves the lemma `TopologicalGroup.existOpenNormalSubgroupSubClopenNhdsOfOne`, which
```

Comment:
I forgot to say earlier: now that this became a lemma and not a definition, it should switch to the lemma naming convention. See https://leanprover-community.github.io/contribute/naming.html.